02-cap-api/srv/ai-service.js

```diff
@@ -33,28 +28,46 @@ export class AIService extends cds.ApplicationService {
     }
 
     /**
-    * Action forwarding prompt to through AI Core provided proxy
-    */
+     * ========================
+     * CHAT COMPLETION (see https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#chat-completions)
+     *
+     * !!! Note: The following action currently supports only non-chat use cases.
+     * !!! For chat based use cases the payload of the messages needs to be adjusted (see https://platform.openai.com/docs/guides/chat)
+     * !!! Also have a look at the markup lang ChatML of OpenAI https://github.com/openai/openai-python/blob/main/chatml.md
+     * ========================
+     */
```

Comment:
i find this part a bit confusing. from the heading it sounds like the action is about chat completion. but then it says it cannot be used for chat use cases. i do not have the full context though.

Reply:
thats okay, it just shows how the chat-api can be leveraged to fulfill non-chat use cases